recipes/libtool/all/conanfile.py

```diff
@@ -92,6 +93,20 @@ def generate(self):
 
         tc = AutotoolsToolchain(self)
 
+        if cross_building(self) and is_msvc(self):
```

Comment:
```suggestion
        if cross_building(self) and (is_msvc(self) or self.settings.compiler == "clang"):
```
Maybe it should be like that?